### ISSUE TITLE
Dead `KaiGen Gothic` link in README

### ISSUE THREAD

**Issue by abelsromero** [chloerei/asciidoctor-pdf-cjk-kai_gen_gothic#8]
I noticed the repository to which the `Kai Gen Gothic` link points does no longer exist.
Googling I found this one https://github.com/minjiex/kaigen-gothic, but I don't know if this is an official continuation.

**Comment by chloerei** [comment]
Yes, it suddenly disappeared, even the accounts are gone, I feel weird. Kai Gen Gothic is base on Google Noto CJK fonts, according to a blog post(disappeared), it's convert by a commerce font editor.

I use this converted font because Prawn(asciidoctor-pdf's dependency) only support `.ttf` format, original Noto fonts is `.otf`.

And because the font's repo is disappeared, I tend to close this theme, but another book project is depending on this theme, so I will keep it until I found a better solution.

If you need a asciidoctor-pdf theme, I recommend follow official guide and create a custom theme: https://github.com/asciidoctor/asciidoctor-pdf/blob/master/docs/theming-guide.adoc#custom-fonts

By the way, I think asciidoctor-pdf base on Prawn is hard to custom styles, and I found http://wkhtmltopdf.org/ is more easy: convert to html first, style by css, then convert to pdf. wkhtmltopdf support system fonts, then I can use original Noto fonts.

**Comment by chloerei** [comment]
I don't know whether minjiex/kaigen-gothic is rename from akiratw/kaigen-gothic, but I don't want to trust this font again, because akiratw suddenly disappeared without description.

**Comment by abelsromero** [comment]
Thanks for the explanation.
I just wanted to point out the link was not working, I have no experience with fonts and I totally agree that changing it for another repo without validation is not a good approach.
